SPHINXsys/src/shared/particle_dynamics/solid_dynamics/solid_dynamics.cpp

```diff
@@ -20,7 +20,7 @@ namespace SPH
 			ContactDynamicsData(solid_body_contact_relation),
 			mass_(particles_->mass_), contact_density_(particles_->contact_density_)
 		{
-			for (size_t k = 0; k != contact_particles_.size(); ++k)
+				for (size_t k = 0; k != contact_particles_.size(); ++k)
```

Comment:
weird space

Reply:
Ok. I will to format the code by clang format in the future merge.